### ISSUE TITLE
Incorrect AssertionError message if test fails

### ISSUE THREAD

**Issue by ilyakharlamov** [llorllale/cactoos-matchers#91]
```
final Iterator<Integer> iter = Arrays.asList(1,2).iterator();
new Assertion<>(
    "Exception should say expected <2> was: <1>",
    () -> iter.next(),
    new IsEqual<>(
        2
    )
).affirm();
```
**What's wrong**:
The error message is:
```Expected: <2>  but was: was <2>```

**Expected result**:
The error message should be like:
```Expected: <2>  but was: <1>```

Tested against 84e2fb9 so it's not fixed in #53 and #86

**Comment by llorllale** [comment]
@0crat in

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/llorllale/cactoos-matchers/issues/91#issuecomment-457922785))

@llorllale Job [#91](https://github.com/llorllale/cactoos-matchers/issues/91) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @ilyakharlamov[/z](https://www.0crat.com/u/ilyakharlamov)

**Comment by 0crat** [comment]
The job [#91](https://github.com/llorllale/cactoos-matchers/issues/91) assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/c494d392-6152-4986-b01e-f9f97e4abe13); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by victornoel** [comment]
@llorllale @ilyakharlamov do we agree that the fact that the term "was" is duplicated here is not the problem but only the fact that the test is executed twice instead of only once?

**Comment by victornoel** [comment]
@0crat wait for #107 to be merged.

**Comment by 0crat** [comment]
> @0crat wait for #107 to be merged. ([here](https://github.com/llorllale/cactoos-matchers/issues/91#issuecomment-480512249))

@victornoel The impediment for [#91](https://github.com/llorllale/cactoos-matchers/issues/91) was registered successfully by @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by victornoel** [comment]
@ilyakharlamov the PR has been merged, I believe you can close this issue

**Comment by 0crat** [comment]
Job [#91](https://github.com/llorllale/cactoos-matchers/issues/91) is not in the agenda of @victornoel[/z](https://www.0crat.com/u/victornoel), can't inspect

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#91](https://github.com/llorllale/cactoos-matchers/issues/91) is now out of scope

**Comment by sereshqua** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)
